src/quo/components/wallet/account_permissions/schema.cljs

```diff
@@ -12,7 +12,7 @@
         [:address [:maybe :string]]
         [:emoji [:maybe :string]]
         [:customization-color {:optional true} [:maybe :schema.common/customization-color]]]]
-      [:token-details {:optional true} [:maybe [:vector required-tokens/?schema]]]
+      [:token-details {:optional true} [:maybe [:sequential required-tokens/?schema]]]
```

Comment:
As pointed out by @ilmotta [here](https://github.com/status-im/status-mobile/pull/18636#discussion_r1467522818), using `:vector` will fail when lazy sequences are passed. @mmilad75, I noticed you [reverted it back to `:vector`](https://github.com/status-im/status-mobile/pull/18707/files#diff-fc2e079cc2a5381f60a93d0cc8c6c85e33ac0705719fa150b17fb0bfa7048791R15). Could you please clarify if there's a specific reason for this choice?

@ilmotta, perhaps we could consider adding this information to the malli guideline section.